src/components/Footer.tsx

```diff
@@ -213,20 +201,78 @@ const Logo = styled.div`
 const FooterLinkGroups = styled.div`
 	display: flex;
 	flex: 1;
+	width: 100%;
+
+	@media(max-width:768px){
+		flex-wrap: wrap;
+	}
 `;
 
 const FooterLinkGroup = styled.div`
-	flex: 1;
+	// flex: 1;
 	display: flex;
 	flex-direction: column;
-	gap: 2rem;
+	gap: 3rem;
+	width: 50%;
+
 	> h5 {
 		margin: 0;
 		font-size: 1.2rem;
 	}
+	@media(max-width:768px){
+		width: 50%;
+		margin-bottom: 4rem;
+	}
+
 `;
 
 const SocialsContainer = styled.div`
 	display: flex;
 	gap: 0.5rem;
 `;
+
+const LogoInfoWrapper = styled.div`
+	display: flex;
+	flex-rap: wrap;
+	gap: 5rem;
+	flex: 1;
+	margin-top: 4rem;
```

Comment:
the space between the edge of hte footer and it's children should be controlled by either the footer or it's container. And in that case it should be `padding`

Reply:
Done